server/@types/index.d.ts

```diff
@@ -144,6 +144,7 @@ export interface EmailUploadValidatorArgs {
   emailFileSelected: boolean
   uploadFailed: boolean
   allowedFileExtensions: string[]
+  actionedByUserId: string
```

Comment:
Minor but I'd just call this `assessedByUserId` so it matches up with the API property name

Reply:
I intentionally didn't call it that as this type is used in 2 places, also when creating the dossier.  Which will presumably have a different API property name